### DISCUSSION TITLE
Support of VanRysel D100 trainer

### DISCUSSION THREAD

**Discussion by blocbul** [cagnulein/qdomyos-zwift#2782] [Q&A] [answered]
Hi,

I have a VanRysel D100 (indentified by VANRYSEL-HT-92c5) and I would like it to be supported.
I tried the Beta APK on Android, it sees the HT in bluetooth but the screen doesn't show any tiles... I suppose it's not communicating correctly with it. 
I've created the bug report as you mention in the "Ask for device support" section.  Where can I share the log with you ?

**Comment by cagnulein** [comment]
Hi, did try to set it under ftms bike setting in the bike options and press
OK?

Let me know!

Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud



Il giorno gio 14 nov 2024 alle 17:24 blocbul ***@***.***> ha
scritto:

**Reply by blocbul** [reply]
Hi, I don't see anything like "ftms bike setting" in the bike options (I have app version 2.14.1)

**Reply by cagnulein** [reply, marked as answer]
2.14?? are you sure it's very old!
where did you download it?
download the last night build https://github.com/cagnulein/qdomyos-zwift/releases/download/nightly/android-debug.apk

**Reply by blocbul** [reply]
OMG... I didn't check the latest releases... 
Thanks a lot, it works perfectly now :)

**Reply by cagnulein** [reply]
that's great! without changing the ftms bike setting right?

**Reply by blocbul** [reply]
right, I didn't changed anything. Just ran the app defaults
